src/lib/Libwin/passwd.c

```diff
@@ -5775,6 +5775,7 @@ get_processowner(DWORD processid, uid_t *puid, char *puname, size_t uname_len, c
 	hProcess = OpenProcess(PROCESS_ALL_ACCESS, FALSE, processid);
 	if ((hProcess == INVALID_HANDLE_VALUE) || (hProcess == NULL)) {
 		log_errf(-1, __func__, "failed in OpenProcess for process: %lu", processid);
+		CloseHandle(hProcess);
```

Comment:
if hProcess is NULL or INVALID_HANDLE_VALUE do you really need to call CloseHandle()? Is that call even valid?

Reply:
Thanks @subhasisb for clarifying the doubt. Will close this PR.